sphinx/source/docs/user_guide/embed.rst

```diff
@@ -120,8 +120,18 @@ Then the corresponding code on the page might look like:
 
     <script>
     fetch('/plot')
-        .then(function(response) { return response.json(); })
-        .then(function(item) { Bokeh.embed.embed_item(item); })
+        .then(function(response) { return response.json() })
+        .then(function(item) { return Bokeh.embed.embed_item(item) })
+    </script>
+
+or with modern syntax:
+
+.. code-block:: html
+
+    <script>
+    const response = await fetch('/plot')
+    const item = await response.json()
+    Bokeh.embed.embed_item(item)
```

Comment:
Why not just update to this throughout and replace the old version?

Reply:
We still support browsers which don't allow this to some or full extent (primarily IE), so it makes sense to show both. `await` syntax is pretty new, so the direct promise-usage version may still be more appealing to an average user. Though I would lean towards making the modern syntax the default.

Reply:
OK, I'm fine with it as is then, but  it would probably be a good idea to define what "modern" means more precisely.   We can work on docs improvements after next week, though, so I will merge this now.

Reply:
i.e. an example of this in use and/or a section in the the users guide somewhere would be valuable to users, I think.